### DISCUSSION TITLE
Set-PnPUserOneDriveQuota Does not set -QuotaWarning

### DISCUSSION THREAD

**Discussion by Dekker024** [pnp/powershell#3927] [Q&A] [answered]
**Reporting an Issue**
If I use :
$StorageWarning = 92160
$StorageMaximum = 102400
Set-PnPUserOneDriveQuota -Account $Owner -Quota $StorageMaximum -QuotaWarning $StorageWarning

The value “$StorageWarning” is not set instead there is a “0” value
The value “$StorageMaximum” is set correctly.
There is no error or warning

**Expected behavior**
Both values should be set
**Steps to reproduce behavior**
$Owner = "someone@tenantdomain"
$TenantAdminUrl = https://tenant-admin.sharepoint.com/
Connect-PnPOnline -Url $TenantAdminURL -Interactive
$StorageWarning = 92160
$StorageMaximum = 102400
Set-PnPUserOneDriveQuota -Account $Owner -Quota $StorageMaximum -QuotaWarning $StorageWarning

Module PnP.PowerShell 2.4.0

**workaround:**
If I use 
Set-PnPTenantSite -Identity $URL -StorageQuota $StorageMaximum -StorageWarningLevel $StorageWarning 

It all works fine.

**Comment by Barbarur** [comment, marked as answer]
Storage is in bytes. You are setting warning at 92kb. I guess the server consider it as 0.

I'm surprise the total storage is set correctly. Maybe there is a default minimum or that was the original storage, so it actually never changed.

Meanwhile Set-PnPTenantSite storage parameters are in Mb, so your command works.
